packages/core/src/logs/DdLogs.ts

```diff
@@ -39,7 +37,9 @@ const isLogWithError = (
 };
 
 class DdLogsWrapper implements DdLogsType {
-    private nativeLogs: DdNativeLogsType = NativeModules.DdLogs;
+    // eslint-disable-next-line global-require, @typescript-eslint/no-var-requires
+    private nativeLogs: DdNativeLogsType = require('../specs/NativeDdLogs')
```

Comment:
if we are compiling for the old architecture (because for Kotlin we have 2 source sets), will it still work?

Reply:
Yes as TurboModules work for both old and new arch (the internal packages of React Native have been using TurboModules since version `0.62` at least, even though they always ran on the old arch)